lark/lark.py

```diff
@@ -73,6 +73,7 @@ class LarkOptions(Serialize):
     edit_terminals: Optional[Callable[[TerminalDef], TerminalDef]]
     import_paths: 'List[Union[str, Callable[[Union[None, str, PackageResource], str], Tuple[str, str]]]]'
     source_path: Optional[str]
+    ordered_sets: bool
```

Comment:
If we want to be consistent with the order of the options, `ordered_sets` could be placed right after `use_bytes`.